hi-ml-histopathology/src/histopathology/configs/classification/DeepSMILECrck.py

```diff
@@ -84,15 +91,15 @@ def cache_dir(self) -> Path:
 
     def setup(self) -> None:
         if self.encoder_type == SSLEncoder.__name__:
-            from InnerEye.ML.configs.histo_configs.run_ids import innereye_ssl_checkpoint_crck_4ws
+            from histopathology.configs.run_ids import innereye_ssl_checkpoint_crck_4ws
             self.downloader = CheckpointDownloader(
-                aml_workspace=get_workspace(),
+                azure_config_json_path=get_workspace(),
```

Comment:
This variable should be named 'aml_workspace' since get_workspace will return a Workspace object rather than a path